src/ostree/ot-builtin-prune.c

```diff
@@ -49,6 +50,7 @@ static GOptionEntry options[] = {
   { "keep-younger-than", 0, 0, G_OPTION_ARG_STRING, &opt_keep_younger_than, "Prune all commits older than the specified date", "DATE" },
   { "static-deltas-only", 0, 0, G_OPTION_ARG_NONE, &opt_static_deltas_only, "Change the behavior of delete-commit and keep-younger-than to prune only static deltas" },
   { "retain-branch-depth", 0, 0, G_OPTION_ARG_STRING_ARRAY, &opt_retain_branch_depth, "Additionally retain BRANCH=DEPTH commits", "BRANCH=DEPTH" },
+  { "only-branch", 0, 0, G_OPTION_ARG_STRING_ARRAY, &opt_only_branches, "Only prune BRANCH (may be specified multiple times)", "BRANCH" },
```

Comment:
Have you thought about making these positional args, e.g. just `ostree prune foo bar`? Seems more natural.

Hmm, although `git prune foo bar` has the opposite meaning:

```
<head>...
    In addition to objects reachable from any of our references, keep objects reachable from listed <head>s.
```

which I find odd, but if in case people are used to that, I'm fine with leaving it as a switch!

Reply:
Mmm...not sure about promoting this to the toplevel quite yet.